core/idl/wallet/common/wallet.djinni

```diff
@@ -265,6 +272,9 @@ OperationQuery = interface +c {
     # Execute query to retrieve operations.
     # @param callback, if execute method succeed, ListCallback object returning a List of Operation objects
     execute(callback: ListCallback<Operation>);
+    # Count operations by type
+    # @param callback, if execute method succeed, ListCallback object returning a List of OperationCount objects
+    count(callback: ListCallback<OperationCount>);
```

Comment:
Could be `const`

Reply:
I can't make  `performCount` const as it uses `_builder` that do not offer const methods by design.

Reply:
Ok then :/